projects/cn0561/src/cn0561.c

```diff
@@ -217,6 +217,12 @@ int main()
 	if (ret != 0)
 		return -1;
 
+	for(int adc_channel = CH0; adc_channel <= CH3; adc_channel++) {
```

Comment:
nit: `for (` . also declaration of `adc_channel` should be placed at the start of the function.